### DISCUSSION TITLE
Sengled E1C- NB7 smart plug not tracking power

### DISCUSSION THREAD

**Discussion by klkohl** [Koenkk/zigbee2mqtt#24084] [Devices]
I have a few of these smart plugs that are not continuously reporting power consumption. They seem to work as expected in ZHA. Any ideas on why these aren't working correctly in Z2M?

**Comment by nvitaterna** [comment]
I also have the same issue. I hadn't used one of these in a while, but set one up again and noticed the reporting configuration seems to do nothing.

I've definitely had these working fine in the past so I'm not sure what's going on.

**Comment by scottrhoyt** [comment]
Noticed the same today as well.
